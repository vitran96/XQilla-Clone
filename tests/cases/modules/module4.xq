import module namespace mod = "mod2" at "lmodule3.xq";

declare function local:f($a) { $a + 1 };

mod:func1(), mod:func2(), local:f(99)
