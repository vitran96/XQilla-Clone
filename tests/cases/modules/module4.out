string and again<b/>and again<b/>100